### PR TITLE
FB Playable Ad documentation

### PR DESCRIPTION
**Pull request by yaustar**
Add FB Playable Ads documentation (videos added to show the playable ads, not sure if needed?)
Screenshot_2021-03-04 Facebook Playable Ad Learn PlayCanvas

Updated REST API page on how to get the parameter values
Screenshot_2021-03-04 REST API Learn PlayCanvas

### CODE REVIEW

File: content/en/user-manual/api/index.md [hunk 1]
```diff
@@ -34,6 +46,26 @@ curl -H "Authorization: Bearer nesgdxhiqe7hylfilr6ss1rds0gq1uj8" https://playcan
 
 Various routes accept a number of parameters. For GET requests if the parameter is not part of the URL, you can pass it as an HTTP query string parameter. For POST, PUT and DELETE requests parameters not included in the URL should be encoded as JSON with a Content-Type of 'application/json'.
 
+There are several common parameters that are used in each endpoint:
+
+### project_id
+
+This is can be found in the URL on the project overview page.
```
Comment (willeastcott) [inline content/en/user-manual/api/index.md:53]:
```suggestion
This can be found in the URL on the project overview page.
```

**Review by willeastcott — APPROVED** [review]

**Comment by yaustar** [comment]
@willeastcott It's ready to be reviewed now 😅

File: content/en/user-manual/publishing/playable-ads/fb-playable-ads.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+title: Facebook Playable Ad
+template: usermanual-page.tmpl.html
+position: 3
+---
+
+PlayCanvas supports the [Facebook Playable Ad][1] formats and requirements via an [official external tool on GitHub][2].
+
+<video loop controls autoplay muted src="/videos/fb-playable-ad-bitmoji-creator.mp4" width="200"> </video> <video loop controls autoplay muted src="/videos/fb-playable-ad-cube-jump.mp4" width="200"></video> <video loop controls autoplay muted src="/videos/fb-playable-ad-flappy-bird.mp4" width="200"></video>
+
+The tool can create both the single 2MB (uncompressed) HTML file and the 5MB (uncompressed) ZIP formats via the configuration options. Full specifications for Facebook Playable Ads can be found on their [help centre][3].
```
Comment (willeastcott) [inline content/en/user-manual/publishing/playable-ads/fb-playable-ads.md:11]: Should this say `5MB (compressed)`?

File: content/en/user-manual/publishing/playable-ads/fb-playable-ads.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+There are some limitations to be aware of with the tool which can be found on the documentation from [GitHub][2].
```
Comment (willeastcott) [inline content/en/user-manual/publishing/playable-ads/fb-playable-ads.md:13]:
```suggestion
There are some limitations to be aware of with the tool which can be found in the documentation from [GitHub][2].
```

File: content/en/user-manual/publishing/playable-ads/fb-playable-ads.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+
+## Example project
+
+[Cube Jump project][5] is ready to exported for the Facebook Playable Ad format and the expected [HTML output can be found here][6].
```
Comment (willeastcott) [inline content/en/user-manual/publishing/playable-ads/fb-playable-ads.md:18]:
```suggestion
The [Cube Jump project][5] is ready to be exported for the Facebook Playable Ad format and the expected [HTML output can be found here][6].
```

File: content/en/user-manual/publishing/playable-ads/fb-playable-ads.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+The minified PlayCanvas Engine code is **~1.2MB** uncompressed and due to the need to encode the asset files into Base64 strings, it adds **~30%** to the size of each asset file.
+
+This means that for a single HTML format, this leaves ~500KB for assets before they are encoded into Base64 strings.
+
+Try to keep images as small as possible in dimensions and use tools like [TinyPNG][4] to reduce file size even further.
+
+
+## How to export
+
+Follow the [setup steps][7] from the read me in the GitHub repo.
```
Comment (willeastcott) [inline content/en/user-manual/publishing/playable-ads/fb-playable-ads.md:36]:
```suggestion
Follow the [setup steps][7] from the readme in the GitHub repo.
```

**Review by willeastcott — APPROVED** [review]
Approved with some comments and suggestions.

File: content/en/user-manual/publishing/playable-ads/fb-playable-ads.md (hunk 2 above)
Comment (yaustar) [reply content/en/user-manual/publishing/playable-ads/fb-playable-ads.md:11]: I'm under the impression that is uncompressed (the docs are really not clear) https://www.facebook.com/business/help/412951382532338?ref=FBB_PlayableAds

```
The total size of uploaded files may not exceed 5 MB.
The number of files inside the bundle may not exceed 100.
The size of index.html file should not exceed 2 MB.
```

**Pull request merged**